src/externs.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @fileoverview This file declares functions that can be declared by external
+ * third party code. Closure compiler advanced optimizations needs this file
+ * to know not to minimize these function names.
+ * @externs
+ */
+
+function shouldPersist(storageInterface, key, value) {}
```

Comment:
Am I forgetting, or was storageInterface was removed from this method?  Practically, I don't think it matters in an externs definition, but ideally it would match the real interface.

Reply:
You aren't forgetting, this file was out of date. Fixed.